js/interface.js

```diff
@@ -307,32 +311,115 @@
     return Fliplet.API.request({
       url: [
         'v1/widgets?include_instances=true&tags=type:menu',
+        '&include_all_versions=true',
         '&appId=' + Fliplet.Env.get('appId'),
-        '&organizationId=' + (Fliplet.Env.get('organizationId') || '')
+        '&organizationId=' + (organizationId || '')
       ].join('')
     }).then(function(response) {
       return Promise.resolve(response.widgets);
     });
   }
 
+  /**
+   * Function to filter organizationMenus by version.
+   * @param {Array} data - array of organizationMenus
+   * @param {Object} latestVersionMenu - current active menu
+   * @returns {Object} latestVersionMenu - object which contains the latest version of current menu package type
+   */
+
+  function sortByVersion(data, latestVersionMenu) {
+    var previousVersion;
+
+    data.forEach(function(menu) {
+      // Compare each record with the previous one by version to find the newest version.
+      if (previousVersion && previousVersion.version !== menu.version) {
+        latestVersionMenu = (previousVersion.version > menu.version) ? previousVersion : menu;
```

Comment:
The way you're using `>` to compare the versions, it will result in `1.9.0` being bigger than `1.10.0`.

![image](https://user-images.githubusercontent.com/290733/104744008-46136180-5744-11eb-9dca-312e88208936.png)

Reply:
@tonytlwu, fixed